lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -388,7 +390,7 @@ sub _generate_workflow_options_gcp {
 
     my $contents = <<"EOCONFIG"
 {
-    "final_workflow_log_dir": "gs://$cromwell_gcp_bucket/build.$build_id/logs",
+    "final_workflow_log_dir": "gs://$cromwell_gcp_bucket/build.$build_id"/logs,
```

Comment:
is this quotation mark supposed to be inside the slash or at the end?

Reply:
This quote is haunting me. I have no idea why it keeps changing back but this will be the fourth time I've fixed it?

Reply:
Ugh it was just my last push didn't go through